src/assets/sass/protocol/components/_notification-bar.scss

```diff
@@ -20,16 +20,11 @@
         padding: 0;
         min-width: $content-sm;
         max-width: $content-md;
-        height: 46px;
     }
 
     p {
         display: inline-block;
-        margin: 0 15px;
-
-        @media #{$mq-sm} {
-            line-height: 46px;
-        }
+        margin: 12px 45px;
```

Comment:
We should use spacing tokens for these values. I'd suggest `margin: $spacing-md $spacing-2xl;` (which is 16px and 48px)

Reply:
Thanks for your guidance @craigcook 
I have made the necessary changes.